File: JavaBasic/170_delete_element.java
// 170	170_delete_element.java	Delete an element from a given position in an array using splice().
import java.util.*;
public class _170_delete_element {
    public static void main(String[] args) {
        List<Integer> list = new ArrayList<>(Arrays.asList(10,20,30,40,50));
        System.out.println("Original: " + list);
        int removed = list.remove(2);
        System.out.println("Removed: " + removed + " | After: " + list);
    }
}

File: JSBASIC/170_delete_element.js
// 170	170_delete_element.js	Delete an element from a given position in an array using splice().

let arr = [10, 20, 30, 40, 50];
let position = 2;
console.log("Original:", arr);
let removed = arr.splice(position, 1);
console.log("Removed element:", removed[0]);
console.log("After deletion:", arr);
